application/dash_application/utility/sidebar_utils.py

```diff
@@ -340,7 +340,7 @@ def get_requested_run_date_range(last_string) -> List[str]:
         return [None, None]
 
 
-def parse_query(query) -> List[str]:
+def parse_query(query) -> Dict[str, Any]:
```

Comment:
Why the change in signature?

Reply:
It was the wrong signature. `queries` is being returned and its a `Dict`